File: solutions/002/_002.java
public class _002 {
    public static void main(String[] args) {
        int[] fibo = new int[36];
        fibo[0] = 0;
        fibo[1] = 1;
        for (int i = 2; i < 34; i++) {
            fibo[i] = fibo[i - 2] + fibo[i - 1];
        }
        int s = 0;
        for (int i = 0; i < 36; i++) {
            if (fibo[i] % 2 == 0)
                s += fibo[i];
        }
        System.out.println(s);
    }
}

File: solutions/002/002.rs
fn main() {
    let mut fibo: [u32; 36] = [0; 36];
    fibo[1] = 1;
    for i in 2..36 {
        fibo[i] = fibo[i - 2] + fibo[i - 1];
    }
    let mut s = 0;
    for i in 2..36 {
        if fibo[i] % 2 == 0 {
            s += fibo[i];
        }
    }
    println!("{}", s);
}
